pkgs/development/libraries/qmltermwidget/default.nix

```diff
@@ -7,8 +7,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     repo = "qmltermwidget";
     owner = "Swordfish90";
-    rev = "v${version}";
-    sha256 = "0ca500mzcqglkj0i6km0z512y3a025dbm24605xyv18l6y0l2ny3";
+    rev = "48274c75660e28d44af7c195e79accdf1bd44963";
```

Comment:
Was #51572 not enough to fix Darwin build of `qmltermwidget` ?

Reply:
Unfortunately not- #51572 fixes the build, but there is another issue once it's running- https://github.com/Swordfish90/cool-retro-term/issues/366